### DISCUSSION TITLE
battery volume can't be showed

### DISCUSSION THREAD

**Discussion by xuyinxiong** [yusifsalam/t490-macos#36] [General]
I put the .aml files in the ACPI/Patch, but the battery can't be recognized. Is there other patch for T490 battery?

**Comment by yusifsalam** [comment]
hmm weird, are you using my EFI as-is or just the battery patch? What model is your T490? Mine is 20N3 for example. 
And no, I don't think there's another patch for the T490 battery specifically, the one included in this repo is the only one I know of and I made it myself.

**Reply by xuyinxiong** [reply]
thanks for your quick response. Yes, I used your EFI and my model is same as yours 20N3. Now the battery volume is 0% and the trackpad isn't drived.

**Reply by yusifsalam** [reply]
Someone else had a similar issue https://github.com/yusifsalam/t490-macos/issues/6 and we weren't able to fix it I'm afraid. Are you on the latest BIOS?

**Reply by xuyinxiong** [reply]
yes,I think I got the same issue

**Reply by xuyinxiong** [reply]
hi Yusifsalam
I replaced ssdt-plug.aml with SSDT-PLUG-_SB.PR00.aml which I downloaded from OC-little. Now battery indicator and touchpad worked.
thanks

**Reply by yusifsalam** [reply]
Glad to hear that and thanks for reporting your solution
